docs/integrators/deep-history-squad.md

```diff
@@ -310,11 +311,34 @@ The reconstruction workspace should look as follows (irrelevant files omitted):
 
 ```
 .
-├── config          # network configuration files
+├── config                                    # network configuration files
```

Comment:
Can add a few extra words (in the paragraph above), for context / for the reader: `When reconstructing a partial history, the workspace ...`.